tests/pytest/oauth/test_views.py

```diff
@@ -99,76 +98,6 @@ def test_oauth_client_or_error_oauth_client(
     mocked_sentry_sdk_module.capture_exception.assert_not_called()
 
 
-@pytest.mark.django_db
-@pytest.mark.usefixtures("mocked_session_flow_uses_claims_verification", "mocked_oauth_client_or_error_redirect__error")
-def test_login_oauth_client_init_error(app_request, mocked_view_analytics_module):
-    result = login(app_request)
-
-    assert result.status_code == 302
-    assert result.url == reverse(routes.OAUTH_SYSTEM_ERROR)
-    mocked_view_analytics_module.error.assert_called_once()
-
-
-@pytest.mark.django_db
-def test_login_no_session_flow(app_request):
-    result = login(app_request)
-
-    assert result.status_code == 200
-    assert result.template_name == TEMPLATE_USER_ERROR
-
-
-@pytest.mark.django_db
-@pytest.mark.usefixtures("mocked_session_flow_uses_claims_verification")
-def test_login(app_request, mocked_oauth_client_or_error_redirect__client, mocked_view_analytics_module):
-    assert not session.logged_in(app_request)
-    mocked_oauth_client = mocked_oauth_client_or_error_redirect__client.return_value
-    # fake a permanent redirect response from the client
-    mocked_oauth_client.authorize_redirect.return_value.status_code = 301
-
-    login(app_request)
-
-    mocked_oauth_client.authorize_redirect.assert_called_with(app_request, "https://testserver/oauth/authorize")
-    mocked_view_analytics_module.started_sign_in.assert_called_once()
-    assert not session.logged_in(app_request)
-
-
-@pytest.mark.django_db
-@pytest.mark.usefixtures("mocked_session_flow_uses_claims_verification")
-def test_login_authorize_redirect_exception(
-    app_request, mocked_oauth_client_or_error_redirect__client, mocked_view_analytics_module, mocked_sentry_sdk_module
-):
-    mocked_oauth_client = mocked_oauth_client_or_error_redirect__client.return_value
-    mocked_oauth_client.authorize_redirect.side_effect = Exception("Side effect")
-
-    result = login(app_request)
-
-    assert result.status_code == 302
-    assert result.url == reverse(routes.OAUTH_SYSTEM_ERROR)
-    mocked_view_analytics_module.error.assert_called_once()
-    mocked_sentry_sdk_module.capture_exception.assert_called_once()
-
-
-@pytest.mark.django_db
-@pytest.mark.usefixtures("mocked_session_flow_uses_claims_verification")
-@pytest.mark.parametrize("status_code", [400, 401, 403, 404, 500, 501, 503])
-def test_login_authorize_redirect_error_response(
-    app_request,
-    mocked_oauth_client_or_error_redirect__client,
-    mocked_view_analytics_module,
-    mocked_sentry_sdk_module,
-    status_code,
-):
-    mocked_oauth_client = mocked_oauth_client_or_error_redirect__client.return_value
-    mocked_oauth_client.authorize_redirect.return_value.status_code = status_code
-
-    result = login(app_request)
-
-    assert result.status_code == 302
-    assert result.url == reverse(routes.OAUTH_SYSTEM_ERROR)
-    mocked_view_analytics_module.error.assert_called_once()
-    mocked_sentry_sdk_module.capture_exception.assert_called_once()
```

Comment:
These are also fine to remove with the same reasoning as above about not wanting to mock an external package's internal implementation.